net/vsftpd/files/vsftpd.init

```diff
@@ -2,12 +2,152 @@
 # Copyright (C) 2006-2011 OpenWrt.org
 
 START=50
+USE_PROCD=1
+BIN=vsftpd
```

Comment:
Shouldnt we rather use hardcoded path as we do for all other packages in its init files instead of relying on (system) PATH?